ui/src/app/base/components/DHCPTable/DHCPTable.tsx

```diff
@@ -70,25 +69,23 @@ const generateRows = (
         name: dhcpsnippet.name,
         description: dhcpsnippet.description,
         enabled,
-        target: machine.fqdn,
+        target: node.fqdn,
         type,
       },
     };
   });
 
 type Props = {
-  systemId: Machine["system_id"];
+  node: Node;
+  nodeType: NodeModels;
```

Comment:
I'm not sure about using this type here. I think the fact that it's giving us the correct string at the moment is purely incidental, though this wouldn't be the case for e.g "pod". IMO it would be safer if we just used `string`, or when the API updates we use the `NodeType` enum that would then map to a display name (or something).

Reply:
Yeah, fair point. Thanks.